### ISSUE TITLE
A problem with random generator on MinGW

### ISSUE THREAD

**Issue by IlyaBizyaev** [weidai11/cryptopp#27]
This code causes a segmentation fault:
```
unsigned int result;
GenerateBlock((byte*)&result, sizeof(unsigned int));
```
Here is the call stack (image, as I cannot copy text from Code::Blocks call stack window):
![Call stack](https://lh3.googleusercontent.com/-c7nVZKuOvKM/Vbsfhu25CGI/AAAAAAAAA0M/5FpsWjZzEqQ/s1600/2015-07-25%2B17-45-31%2B%25D0%25A1%25D0%25BA%25D1%2580%25D0%25B8%25D0%25BD%25D1%2588%25D0%25BE%25D1%2582%2B%25D1%258D%25D0%25BA%25D1%2580%25D0%25B0%25D0%25BD%25D0%25B0.png)
This problem may be caused by [issue 26](https://github.com/weidai11/cryptopp/issues/26).

**Comment by noloader** [comment]
We have not been able to reproduce your results.

Please try the latest, proposed Crypto++ candidate for release. As of this writing, it is RC1. The announcement with download links and checksums is available at [Crypto++ 5.6.3 Release Candidate available (RC1)](https://groups.google.com/d/msg/cryptopp-users/3UnTZ9TD-ug/BctTMbrYBwAJ).

**Comment by noloader** [comment]
I believe we cleared this when around the 5.6.3 release.
